readthedocs/core/apps.py

```diff
@@ -0,0 +1,12 @@
+from django.apps import AppConfig
+
+
+class CoreAppConfig(AppConfig):
+    name = 'readthedocs.core'
+    verbose_name = 'Core'
+
+    def ready(self):
+        if hasattr(self, 'already_run'):
+            return
+        self.already_run = True
```

Comment:
I've seen this in a few places. What is the purpose of this check? `ready` should only be called once, afaik. Is there a case where it is called multiple times?